src/ui/qgsnewgeopackagelayerdialogbase.ui

```diff
@@ -478,28 +478,62 @@
    <header>qgsscrollarea.h</header>
    <container>1</container>
   </customwidget>
+  <customwidget>
+   <class>QgsCollapsibleGroupBox</class>
+   <extends>QWidget</extends>
+   <header>qgscollapsiblegroupbox.h</header>
+  </customwidget>
  </customwidgets>
  <tabstops>
   <tabstop>scrollArea</tabstop>
   <tabstop>mDatabaseEdit</tabstop>
   <tabstop>mSelectDatabaseButton</tabstop>
   <tabstop>mTableNameEdit</tabstop>
-  <tabstop>mLayerIdentifierEdit</tabstop>
-  <tabstop>mLayerDescriptionEdit</tabstop>
-  <tabstop>mFeatureIdColumnEdit</tabstop>
   <tabstop>mGeometryTypeBox</tabstop>
-  <tabstop>mGeometryColumnEdit</tabstop>
+  <tabstop>mGeometryWithZCheckBox</tabstop>
+  <tabstop>mGeometryWithMCheckBox</tabstop>
   <tabstop>mCrsSelector</tabstop>
-  <tabstop>mCheckBoxCreateSpatialIndex</tabstop>
   <tabstop>mFieldNameEdit</tabstop>
   <tabstop>mFieldTypeBox</tabstop>
   <tabstop>mFieldLengthEdit</tabstop>
   <tabstop>mAddAttributeButton</tabstop>
   <tabstop>mAttributeView</tabstop>
   <tabstop>mRemoveAttributeButton</tabstop>
+  <tabstop>mLayerIdentifierEdit</tabstop>
+  <tabstop>mLayerDescriptionEdit</tabstop>
+  <tabstop>mFeatureIdColumnEdit</tabstop>
+  <tabstop>mGeometryColumnEdit</tabstop>
+  <tabstop>mCheckBoxCreateSpatialIndex</tabstop>
  </tabstops>
  <resources>
   <include location="../../images/images.qrc"/>
+  <include location="../../images/images.qrc"/>
```

Comment:
What's happening here, a few too many? 😄

Reply:
It's a bug in qt creator. They are everywhere now, so I figure we just wait to qt fixes it.